### ISSUE TITLE
Windows build failures

### ISSUE THREAD

**Issue by uzadow** [libavg/libavg#605]
The windows build fails with this message:
```
"C:\devel\buildbot\build\build\win\player\player.vcxproj" (default target) (19) ->
(ClCompile target) -> 
  C:\devel\buildbot\build\build\src\player\GPUImage.cpp : fatal error C1083: Cannot open compiler generated file: 'Release\GPUImage.obj': Permission denied [C:\devel\buildbot\build\build\win\player\player.vcxproj]
  cl : Command line error D8040: error creating or communicating with child process [C:\devel\buildbot\build\build\win\player\player.vcxproj]
```

**Comment by oxullo** [comment]
This specific error is fixed now, the reason why it happened it's unclear to me. Not the first time it though it seems that the FS permissions set for some files generated during the build don't allow the buildbot user to delete them.

**Comment by oxullo** [comment]
As discovered after another incident, the real fix to this issue was 038cc37672b29b4f2b8054dc72a0308278de42b5
